stable/weave-scope/charts/weave-scope-frontend/templates/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "weave-scope-frontend.fullname" . -}}
+{{- $ingressPaths := .Values.ingress.paths -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app.kubernetes.io/name: {{ include "weave-scope-frontend.name" . }}
```

Comment:
use the same labels for the other templates https://github.com/helm/charts/blob/8802c01a8de922e22db88d8ab0dff689343479ef/stable/weave-scope/charts/weave-scope-frontend/templates/deployment.yaml#L15

Reply:
@cesarokuti PTAL?